src/amo/components/AddonReviewListItem/index.js

```diff
@@ -310,7 +316,8 @@ export function mapStateToProps(
   }
   return {
     editingReview,
-    isAuthenticated: isAuthenticated(state),
+    siteUserIsAuthenticated: isAuthenticated(state),
+    siteUserHasReplyPerm: hasPermission(state, ADDONS_EDIT),
```

Comment:
Maybe `siteUserHasReplyPermission` would be better, but it soooo long..

Reply:
> Maybe `siteUserHasReplyPermission` would be better

I like *has...perm* the best https://i.pinimg.com/736x/e4/91/fa/e491fabe4365fb2b62e193601c44c4d7--hair-perms-historical-hairstyles.jpg